src/openforms/dmn/base.py

```diff
@@ -2,6 +2,8 @@
 from dataclasses import dataclass
 from typing import Any
 
+from django_camunda.dmn.datastructures import DMNIntrospectionResult
```

Comment:
I don't think we should use camunda-specific data shapes for the generic implementation.

There should be a decoupled base class/protocol describing what data should be returned by a plugin, since you have an endpoint that is plugin-agnostic.

So you'll either need to define a base class or a protocol to prescribe what needs to be returned by a plugin.